.infrastructure/helm-chart/todoapp/charts/mysql/templates/ns.yml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.mysql.namespace }}
```

Comment:
![image](https://github.com/user-attachments/assets/5dc3fd99-38dc-44a2-ad67-331476f74400)
Add Eof(End-of-file), need to add blank line at the end